src/actions/breakpoints/tests/__snapshots__/syncing.js.snap

```diff
@@ -18,11 +20,34 @@ Object {
       "sourceUrl": "http://localhost:8000/gen.js",
     },
     "hidden": false,
+    "id": Object {
+      "astLocation": Object {
+        "name": undefined,
+        "offset": Object {
+          "line": 3,
+        },
+      },
+      "condition": null,
+      "disabled": false,
+      "generatedLocation": Object {
+        "column": undefined,
+        "line": 3,
+        "sourceId": "gen.js",
+        "sourceUrl": "http://localhost:8000/gen.js",
+      },
+      "hidden": false,
+      "location": Object {
+        "column": undefined,
+        "line": 3,
+        "sourceId": "magic.js",
+        "sourceUrl": "http://localhost:8000/magic.js",
+      },
+    },
```

Comment:
this is a lot of new data to be storing...

Reply:
@jasonLaster we are already looking into it.
@bomsy is helping me out finding what's going on here